thinc/backends/ops.py

```diff
@@ -314,14 +382,14 @@ def unpad(self, padded: Array3d, lengths: List[int]) -> List2d:
             output.append(padded[i, :length])
         return cast(List2d, output)
 
-    def list2padded(self, seqs: List[Floats2d]) -> Padded:
+    def list2padded(self, seqs: List2d) -> Padded:
```

Comment:
For these cases where we relax the signature, we should also add tests for the newly-added types (like `List[Ints2d]` here). Would be a nice follow-up PR.